### ISSUE TITLE
The image suddenly stopped working when updating

### ISSUE THREAD

**Issue by zacekjakub** [acemod/docker-reforger#43]
**Describe the bug**
If I try to redeploy the image now, I get platform error.

**To Reproduce**
It does on all of my servers

**Expected behavior**
SHould be just redeployed

**Screenshots / Output**
Loading Steam API...OK
Connecting anonymously to Steam Public...OK
Waiting for client config...OK
Waiting for user info...OK
ERROR! Failed to install app '1874900' (Invalid platform)

**Comment by suhaibmalik** [comment]
Probably related to this: https://steamdb.info/app/1874900/history/?changeid=26374567

Ref: https://github.com/ValveSoftware/steam-for-linux/issues/11522#issuecomment-2506761098

**Comment by hafometh88** [comment]
yeah so the situation rn is that it works for the main application with steamId '1874900', but not for the experimental application with steamId '1890870'

Bacon shared on the Arma Reforger discord that he uses this workaround to make it work:
 
![Image](https://github.com/user-attachments/assets/e2d348c1-c93f-4f6b-80d3-fb4b21de2eab)

```
RUN /steamcmd/steamcmd.sh +login anonymous +quit
RUN /steamcmd/steamcmd.sh +force_install_dir /home/reforger +login anonymous +@sSteamCmdForcePlatformType windows +app_update 1890870 validate +quit
RUN /steamcmd/steamcmd.sh +force_install_dir /home/reforger +login anonymous +@sSteamCmdForcePlatformType linux +app_update 1890870 validate +quit
```

I'll try to implement this somehow into this project because creating another image for experimental seems cumbersome, but no promises.

**Comment by hafometh88** [comment]
I made a PR #49 that should implement this bypass.
